app/templates/components/session-card.hbs

```diff
@@ -2,7 +2,7 @@
   {{#unless device.isMobile}}
     <div class="ui card three wide computer six wide tablet column">
       <a class="image" href="{{href-to 'my-sessions.view' session.id}}">
-        {{widgets/safe-image src='assets/images/landing.jpg'}}
+        {{widgets/safe-image src=(if session.event.thumbnailImageUrl session.event.thumbnailImageUrl session.event.originalImageUrl)}}
```

Comment:
keep `assets/images/landing.jpg` in the else hook. Under no circumstance, we want to be loading the original big image anymore. Thanks!